astropy/visualization/wcsaxes/tests/test_images.py

```diff
@@ -1160,15 +1231,14 @@ def test_1d_plot_1d_sliced_low_level_wcs(
     """
     Test that a SLLWCS through a coupled 2D WCS plots as line OK.
     """
-    fig = plt.figure()
+    fig = Figure()
     ax = fig.add_subplot(1, 1, 1, projection=spatial_wcs_2d_small_angle[slices])
     (lines,) = ax.plot([10, 12, 14, 12, 10], "-o", color="orange")
 
     # Draw to trigger rendering the ticks.
-    plt.draw()
-
-    assert ax.coords[bottom_axis].get_ticks_position() == ["b", "#"]
+    fig.canvas.draw()
 
+    # assert ax.coords[bottom_axis].get_ticks_position() == ["b", "#"]
```

Comment:
Why is this commented?
```suggestion
    assert ax.coords[bottom_axis].get_ticks_position() == ["b", "#"]
```

Reply:
hum, it's been a while, so maybe I don't remember well, but IIRC this assertion *doesn't* pass with my refactor, but the image *still* remains identical, so I assumed it wasn't crucial to the test. I'll push your suggestion as a separate commit after CI runs again to check my recollection.

Reply:
update: https://app.circleci.com/pipelines/github/astropy/astropy/25104/workflows/b7540ef7-446b-495b-b215-81c3309ce91d/jobs/132936